src/applications/financial-status-report/components/householdExpenses/CreditCardBillSummary.jsx

```diff
@@ -32,10 +33,6 @@ const CreditCardBillSummary = ({
   const { expenses } = formData;
   const { creditCardBills } = expenses || [];
 
-  useEffect(() => {
-    clearJobIndex();
-  }, []);
-
```

Comment:
I removed this useEffect because it didn't seem to be related to the CreditCardBills component, let me know if I need to add it back in